.idea/JMRI.iml

```diff
@@ -52,6 +52,5 @@
       </library>
     </orderEntry>
     <orderEntry type="library" name="lib" level="project" />
-    <orderEntry type="library" name="Python 2.7 interpreter library" level="application" />
```

Comment:
Changing an IDE control file will affect other developers.  Is this change necessary?